src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2217,7 +2217,35 @@ private static string[] ParseArgv(ProcessStartInfo psi)
         {
             var argvList = new List<string>();
             argvList.Add(psi.FileName);
-            argvList.AddRange(psi.Arguments.Split(' '));
+
+            var argsToParse = psi.Arguments.Trim();
+            var argsLength = argsToParse.Length;
+            for (int i=0; i<argsLength; )
+            {
+                var iStart = i;
+         
+                switch (argsToParse[i])
+                {
+                    case '"':
+                        // Special case for arguments within quotes
+                        // Just return argument value within the quotes
+                        while ((++i < argsLength) && argsToParse[i] != '"') { };
```

Comment:
Single quotes are not supported and not needed.  It is something we could add if we need it in the future.  Argument validation is not intended to be performed here.  This just parses space delimited strings with a special case for double quoted arguments.

I just tried to make sure it doesn't blow up with malformed strings and cause an access violation or null reference exception.